### DISCUSSION TITLE
You should add more example for how to use the native component

### DISCUSSION THREAD

**Discussion by 0xroylee** [airbnb/goji-js#156] [Show and tell]
There are serveral components in the source code but only few components are showed in the example.

![螢幕截圖 2022-03-30 上午3 41 30](https://user-images.githubusercontent.com/53046899/160694041-38712ea1-29cb-4dae-9033-90d32b7194db.png)

**Comment by malash** [comment]
Agree, there needed to be detailed documentation about avialiable components.

Ref: https://github.com/airbnb/goji-js/issues/178#issuecomment-1328294295
